src/main/java/uk/gov/companieshouse/api/accounts/service/impl/AbstractServiceImpl.java

```diff
@@ -34,19 +35,26 @@
 
     private GenericTransformer<T, U> genericTransformer;
 
+    private MongoRepository<V, String> parentMongoRepository;
```

Comment:
as discussed, the service shouldn't deal with the parent mongo repository directly, it should use the parent service layer to handle patching links into parent resources